po/pt_BR.po

```diff
@@ -0,0 +1,640 @@
+
+#: print.go:359
+msgid "Total Size occupied by packages: %s"
+msgstr "Espaço total ocupado por pacotes: %s"
+
+#: print.go:357
+msgid "Total foreign installed packages: %s"
+msgstr "Espaço total ocupado por pacotes não nativos: %s"
+
+#: print.go:356
+msgid "Total installed packages: %s"
+msgstr "Total de pacotes instalados: %s"
+
+#: install.go:486
+msgid "Try to build them anyway?"
+msgstr "Tentar construir eles mesmo assim?"
+
+#: print.go:290
+msgid "URL"
+msgstr "URL"
+
+#: print.go:288
+msgid "Version"
+msgstr "Versão"
+
+#: print.go:301
+msgid "Votes"
+msgstr "Votos"
+
+#: print.go:354
+msgid "Yay version v%s"
+msgstr "Versão do Yay v%s"
+
+#: install.go:581 install.go:656 install.go:663
+msgid "[N]one"
+msgstr "[N]Nenhum"
+
+#: keys.go:122
+msgid ""
+"\n"
+"%s %s, required by: %s"
+msgstr ""
+"\n"
+"%s %s, requeridos por: %s"
+
+#: clean.go:94
+msgid ""
+"\n"
+"Build directory:"
+msgstr ""
+"\n"
+"Diretório de Build:"
+
+#: callbacks.go:56 print.go:628
+msgid ""
+"\n"
+"Enter a number (default=1): "
+msgstr ""
+"\n"
+"Digite um número (padrão=1): "
+
+#: depCheck.go:153
+msgid ""
+"\n"
+"Inner conflicts found:"
+msgstr ""
+"\n"
+"Conflitos internos encontrados:"
+
+#: depCheck.go:167
+msgid ""
+"\n"
+"Package conflicts found:"
+msgstr ""
+"\n"
+"Conflitos de pacote encontrados:"
+
+#: install.go:268 install.go:308 install.go:487 install.go:591 install.go:674
+msgid "aborting due to user"
+msgstr "abortando devido ao usuário"
+
+#: install.go:513
+msgid "cannot find package name: %v"
+msgstr "nome de pacote não encontrado: %v"
+
+#: install.go:1046 install.go:1118
+msgid "could not find PKGDEST for: %s"
+msgstr "não foi possível encontrar PKGDEST para: %s"
+
+#: install.go:790
+msgid "editor did not exit successfully, aborting: %s"
+msgstr "editor não saiu com sucesso, abortando: %s"
+
+#: download.go:84 download.go:109
+msgid "error cloning %s: %s"
+msgstr "erro ao clonar %s: %s"
+
+#: install.go:922
+msgid "error downloading sources: %s"
+msgstr "erro ao baixar as fontes: %s"
+
+#: query.go:234
+msgid "error during AUR search: %s"
+msgstr "error during AUR search: %s"
+
+#: download.go:96 download.go:121
+msgid "error fetching %s: %s"
+msgstr "erro ao buscar %s: %s"
+
+#: install.go:331 install.go:440
+msgid "error installing repo packages"
+msgstr "erro ao instalar pacotes do repositório"
+
+#: install.go:1030 install.go:1070 install.go:1081 install.go:1093
+msgid "error making: %s"
+msgstr "erro ao construir: %s"
+
+#: download.go:135
+msgid "error merging %s: %s"
+msgstr "erro ao mesclar %s: %s"
+
+#: download.go:89 download.go:114
+msgid "error reading %s"
+msgstr "erro ao ler %s"
+
+#: install.go:73
+msgid "error refreshing databases"
+msgstr "error ao recarregar base de dados"
+
+#: clean.go:217 download.go:130
+msgid "error resetting %s: %s"
+msgstr "error ao resetar %s: %s"
+
+#: main.go:111
+msgid "failed to create BuildDir directory '%s': %s"
+msgstr "failed to create BuildDir directory '%s': %s"
```

Comment:
@agendator I think you forgot to translate this message here.

Reply:
Thanks for spotting it!